### ISSUE TITLE
No such file or directory, when initializing API tests

### ISSUE THREAD

**Issue by cebe** [Codeception/Codeception#4485]
#### What are you trying to achieve?

Init codeception api testing following this guide: http://codeception.com/quickstart

#### What do you get instead?

```
PHP Warning:  file_put_contents(tests/_support/_generated/.gitignore): failed to open stream: No such file or directory in /home/cebe/dev/api/vendor/codeception/codeception/src/Codeception/InitTemplate.php on line 204

Warning: file_put_contents(tests/_support/_generated/.gitignore): failed to open stream: No such file or directory in /home/cebe/dev/api/vendor/codeception/codeception/src/Codeception/InitTemplate.php on line 204
```

Steps to reproduce:

```
$ composer require "codeception/codeception" --dev
Using version ^2.3 for codeception/codeception
...

$ vendor/bin/codecept init api
Let's prepare Codeception for REST API testing

? Where tests will be stored? (tests) 
? Start url for tests (http://localhost/api) http://localhost:8088                                  
PHP Warning:  file_put_contents(tests/_support/_generated/.gitignore): failed to open stream: No such file or directory in /home/cebe/dev/api/vendor/codeception/codeception/src/Codeception/InitTemplate.php on line 204

Warning: file_put_contents(tests/_support/_generated/.gitignore): failed to open stream: No such file or directory in /home/cebe/dev/api/vendor/codeception/codeception/src/Codeception/InitTemplate.php on line 204
> Created test directories inside at tests
> Api helper has been created in tests/_support/Helper
> ApiTester actor has been created in tests/_support
> Actions have been loaded
> Created global config codeception.yml inside the root directory
> Created a demo test ApiCest.php

 INSTALLATION COMPLETE 

Next steps:
1. Edit tests/ApiCest.php to write first API tests
2. Run tests using: codecept run

Happy testing!
```

### Details

* Codeception version: 2.3.5
* PHP Version: 5.6.30-0+deb8u1
* Operating System: Debian 8.9
* Installation type: Composer

Seems at this point the directory should be created if it does not exist:

https://github.com/Codeception/Codeception/blob/1dca8f38e05c94e4ec622c1ba9b4ff02bc0b630d/src/Codeception/InitTemplate.php#L204

**Comment by Naktibalda** [comment]
Fixed by #4449
Not released yet.

@DavertMik it is about time to make a new release.

**Comment by Naktibalda** [comment]
@cebe if you want it now, you can change a version of composer to "2.3.x-dev" in composer.json and `composer update`
